install_config/master_node_configuration.adoc

```diff
@@ -897,13 +882,12 @@ rules:
     - group: "" # core API group
     - group: "extensions" # Version of group should NOT be included.
 ----
-<1> There are four possible levels every event can be logged at:
-`None` - Do not log events that match this rule.
-`Metadata` - Log request metadata (requesting user, time stamp, resource, verb, etc.), but
-not request or response body. This is the same level as the one used in basic
-audit.
-`Request` - Log event metadata and request body, but not response body.
-`RequestResponse` - Log event metadata, request, and response bodies.
+<1> There are four possible levels every event can be logged at: `None` - Do not
+log events that match this rule. `Metadata` - Log request metadata (requesting
+user, time stamp, resource, verb, etc.), but not request or response body. This
+is the same level as the one used in basic audit. `Request` - Log event metadata
+and request body, but not response body. `RequestResponse` - Log event metadata,
+request, and response bodies.
```

Comment:
This might work better as a list, using a `+` character between each line:
There are four possible levels every event can be logged at: 
+
* `None` - Do not log events that match this rule. 
+
* `Metadata` - Log request metadata (requesting user, time stamp, resource, verb, etc.), but not request or response body. This is the same level as the one used in basic audit. 
+
* `Request` - Log event metadata and request body, but not response body. 
+
* `RequestResponse` - Log event metadata, request, and response bodies.

Reply:
Thank you, I was scratching my head how to do it properly. Will update right away.